nncf/common/pruning/model_analysis.py

```diff
@@ -167,6 +167,9 @@ def propagate_can_prune_attr_up(self):
         for node in reversed_sorted_nodes:
             # Check all output nodes accept_pruned_input attribute
             out_nodes = self.graph.get_next_nodes(node)
+            if out_nodes == []:
+                self.can_prune[node.node_id] =False
```

Comment:
Could you please auto-format with pylint?
It seems that this package uses https://github.com/openvinotoolkit/nncf/blob/develop/.pylintrc.
Not only for this line `=False`.